### PR TITLE
allow for beta tag pulp cloning

### PR DESCRIPTION
**Pull request by jlsherrill**
and other fixes

### CODE REVIEW

File: lib/tool_belt/pulp_repository_updater.rb [hunk 1]
```diff
@@ -34,21 +35,21 @@ def compare_repos(os_version, client = false)
...
         index = File.read('index.html')
         rpms = index.scan(/href=".*.rpm"/).collect { |link| link.split('href=')[1].gsub('"', '') }
         rpms.each do |rpm|
-          syscall("wget #{pulp_repo(os_version)}#{rpm}")
+          @systools.syscall("wget #{pulp_repo(os_version)}#{rpm}")
         end
-        syscall("sudo yum -y install createrepo") unless syscall('rpm -q createrepo')[1]
-        syscall("createrepo .")
+        @systools.syscall("sudo yum -y install createrepo") unless @systools.syscall('rpm -q createrepo')[1]
+        @systools.syscall("createrepo .")
```
Comment (ehelms) [inline lib/tool_belt/pulp_repository_updater.rb]: Should execute be used here instead of syscall to have dry run support?

**Comment by jlsherrill** [comment]
@ehelms updated it

File: lib/tool_belt/pulp_repository_updater.rb (hunk 1 above)
Comment (jlsherrill) [inline lib/tool_belt/pulp_repository_updater.rb]: Actually it should be used here, as you need to create repo in order to compare the repos

**Comment by ehelms** [comment]
Thanks @jlsherrill

**Pull request merged**